### DISCUSSION TITLE
Como pegar a referência de transação pra usar o `get_account_investments_details`

### DISCUSSION THREAD

**Discussion by Ch3did** [andreroggeri/pynubank#350] [Q&A] [answered]
Estou tentando fazer o request para pegar os detalhes de uma transação, mas não encontro a referência dessa transação para usar no `get_account_investments_details`. Alguem consegue me dar uma luz?

**Comment by andreroggeri** [comment, marked as answer]
`get_account_investments_details` não recebe nenhum parâmetro. O que você está tentando fazer ?

**Reply by Ch3did** [reply]
Passei o método errado, seria o `get_card_statement_details`... Quero pegar os detalhes de uma transação. A Ideia ia ser analisar a resposta desse método pra criar uma lógica de seleção de estabelecimentos, relacionando as transações... Eu ja estou utilizando o `get_card_statements` para pegar todas as transações, agr queria analisar transação a transação pra identificar os estabelecimentos

**Reply by Ch3did** [reply]
Mas n estou conseguindo fazer um request para `get_card_statement_details`, ele me a "referência da transação", que n sei aonde pegar

**Reply by andreroggeri** [reply]
Você passa o objeto inteiro que foi recuperado no `get_card_statements`.

Exemplo:

```python
statements = nu.get_card_statements()
detail = nu.get_card_statement_details(statements[0]) # Recuperar os detalhes da primeira transação

print(detail)
```

**Reply by Ch3did** [reply]
Mto obrigado!!!
